lib/src/opentdf.ts

```diff
@@ -286,6 +297,7 @@ export type TDFReader = {
 // SDK for dealing with OpenTDF data and policy services.
 export class OpenTDF {
   // Configuration service and more is at this URL/connectRPC endpoint
+  readonly platformUrl: string;
   readonly policyEndpoint: string;
```

Comment:
What is the difference between `platformUrl` and `policyEndpoint`?

Reply:
That's a great question.  At this point the difference is mostly semantics.

`policyEndpoint` is derived by subtracting `/kas` from the `kasEndpoint`
https://github.com/opentdf/web-sdk/blob/bdb5ba6402503c27016e26f38d6208924e6b6f74/lib/tdf3/src/client/index.ts#L309

Looking at the code base, `policyEndpoint` is guessed in a few places, but I can't seem to find a place where it is used.

If it's safe to assume that `policyEndpoint == platformUrl` I can update this PR to drop `platformUrl` and use `policyEndpoint` instead.  

With this current PR the main difference between the two is that `platfromUrl` is NEVER guessed, it needs to be set explicitly

Reply:
@dmihalcik-virtru Followup sounds perfect on this.

`platformEndpoint` does not exactly fit for what it would be because it's not an endpoint. It's the URL where the platform is hosted. And we also have `policyEndpoint` which points to the same thing as `platformUrl` except policyEndpoint sometimes is guessed by removing `/kas` sufix from the `kasEndpoint`

I do agree, we need to cleanup  the public api for the  OpenTDF  classs